timeservice.go

```diff
@@ -560,9 +615,7 @@ func main() {
 			if authMode != "" && authMode != authModeNTS {
 				exitWithUsage()
 			}
-			if authMode == authModeNTS && ntskeServerName == "" {
-				exitWithUsage()
-			}
+			ntskeServerName := strings.Split(remoteAddrStr, ",")[1]
```

Comment:
`ntskeServer` instead of `ntskeServerName` would be better now, I think.

Reply:
Done